Assets/FullSerializer/Source/Reflection/fsMetaType.cs

```diff
@@ -66,6 +66,11 @@ public class fsMetaType {
                 PropertyInfo property = member as PropertyInfo;
                 FieldInfo field = member as FieldInfo;
 
+                // Skip properties if we don't want them, to avoid the cost of checking attributes.
```

Comment:
That's right - in my use case, switching out an old serialiser for FullSerializer, the old library serialised private fields by default and didn't handle properties. FullSerializer was trying to serialise my properties which was bloating the serialised data size. Adding this option (and using `fsMemberSerialization.OptOut`) was the cheapest (in terms of code changes) way to get parity with what I had before.